### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by emilytrau**
## Description

 This adds support for running the DAShboard with Docker.

## Screenshots

N/A

## Steps to Test

Follow the new setup instructions in the README.

### CODE REVIEW

**Review by hallgchris — COMMENTED** [review]
Noob question here - how do we connect dashboard running in a docker container to a local MQTT broker?

**Comment by emilytrau** [comment]
@hallgchris I added a note about this to the README section. On Linux you would use the address of the `docker0` interface instead of `localhost`

**Comment by hallgchris** [comment]
Oh yeah, I gotcha. I got it to work by modifying the hardcoded address on line 85 of `sockets.js`. Did you have a neater way doing it or is that the best we have for now?

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+version: '3.4'
+
+services:
+  client:
+    image: docker.pkg.github.com/monash-human-power/dashboard/client
```
Comment (hallgchris) [inline docker-compose.yml:5]: What does this bit do?

**Review by hallgchris — CHANGES_REQUESTED** [review]
Docker stuff looks good to me! But as discussed in the meeting yesterday, I reckon we need some way to easily connect the dashboard to a MQTT broker, cos I'd be hesitant to have the master branch require modification to make it usable whenever someone checks it out

**Pull request closed without merging**